src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -215,8 +265,7 @@ When the dataset is coming from HuggingFace, you can specify the following
 options:
 * `--input-dataset {openorca,cnn_dailymail}`: HuggingFace dataset to use for
   benchmarking.
-* `--num-prompts <int>`: The number of unique prompts to generate as stimulus,
-  >= 1.
+* `--num-prompts <int>`: The number of unique prompts to generate as stimulus, >= 1.
```

Comment:
do we not have general options in their own section?

Reply:
They do, this section just talks about the input related options.